### ISSUE TITLE
css image filter makes images disappear

### ISSUE THREAD

**Issue by iggmack** [patrickkunka/mixitup#218]
The css grayscale filter makes the images in two of the columns disappear in the latest version of the plugin. Any ideas why?

http://www.selanderax.se/projekt/

**Comment by patrickkunka** [comment]
Are you applying the filters directly to mixitup targets? If so, have you tried applying the filter to a nested child element instead?

**Comment by iggmack** [comment]
Hello and thanks for the reply. I am not applying the filters directly on the targets, just on the images.

#projekt .panel img{
  -webkit-filter: grayscale(1);
  filter: grayscale(1);
  position: static;
}

#projekt .panel:hover img{
  -webkit-filter: grayscale(0);
  filter: grayscale(0);
  position: static;
}

#project is the mixitup target

**Comment by patrickkunka** [comment]
Hi - I was able to recreate the issue but this doesn't appear to be anything to do with MixItUp. With the `display: none; rule` removed from the css and the `style="display: inline-block;"` attribute removed form the markup, the issue persists, so it's more likely this is a chrome rendering bug of some sort concerning CSS3 columns and filters. If we remove the column settings on the parent container, the appears disappears.

**Comment by iggmack** [comment]
The strange thing is that it worked before. Probably on an earlier version of the plugin?

/E

**Comment by patrickkunka** [comment]
Can you confirm that your column layout with grayscale filters works without mixitup running on the page?

**Comment by jonathanbell** [comment]
I found this page because I was styling a page with CSS3 column layout and images with filters (greyscale) applied. All of the images are not displayed (take up the normal area on screen) in Chrome if they are inside a CSS3 column layout with a filter applied.

**Comment by simgooder** [comment]
I am using the blur() filter, without columns and still getting the disappearing images.

**Comment by Josecc** [comment]
Hey! I'm having the same issue and am no using the plugin either. Adding position relative to the image object did not work.
 What I'm doing is blurring the image on hover, and thats when it *disappears*. I also noticed that adding a shadow would add two lines to the bottom left corner--the two top shadows for the two right columns. Adding a top padding I was able to fix this, but I still have the same issue with the blurring.
